.azure/pipelines/site-extensions.yml

```
@@ -0,0 +1,35 @@
+trigger:
+  branches:
+    include:
+    - release/2.2
+
+phases:
+- phase: SiteExtensions
+  variables:
+    CI: true
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+    # This variable is required by MicroBuildSigningPlugin to determine permissions for codesigning.
+    TeamName: AspNetCore
+
+    # SignType = { test, real }
+    # This is prefixed underscore because variables automatically become environment variables (and therefore MSBuild properties),
+    # and this one was causing issues in MSBuild projects which use the $(SignType) MSbuild prop.
+    _SignType: test
+
+  steps:
+  # - task: MicroBuildSigningPlugin@1
```

You'll need to uncomment this for official builds. You'll also need this task at the end of the build:
```yml
  - task: MicroBuildCleanup@1
    displayName: Cleanup MicroBuild tasks
    condition: always()
```

---

you can also do this to exclude the tasks and other variables in PRs and public builds.

```suggestion
  ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
    - task: MicroBuildSigningPlugin@1
```

---

I don't think it's required, it would never run in public after getting merged.